### PR TITLE
fix: sonarcloud security hotspots

### PR DESCRIPTION
**Pull request by isabellaahc**
This PR includes total reviews about sonarcloud security hotspots.

Changelog: 

- [x] Add rel=noopener on modal links 
- [x] Add https to virtuemart versions URL
- [x] Add script bootstrap url with integrity attribute
- [x] Replace session_id function by session_regenerate_id on emptyCart function
- [x] Add sonarcloud properties file to exclude docker folder and log4php library from sonar cloud analysis

Evidence install plugin and modal status: 
Captura de pantalla 2023-08-24 a la(s) 18 21 00
Captura de pantalla 2023-08-24 a la(s) 18 21 23
Captura de pantalla 2023-08-24 a la(s) 18 22 20

https://github.com/TransbankDevelopers/transbank-plugin-virtuemart-webpay-rest/assets/16402208/fd225c8d-3dff-448c-9b02-264ed6c59ee8

https://github.com/TransbankDevelopers/transbank-plugin-virtuemart-webpay-rest/assets/16402208/8441cf28-fe75-499a-8dee-2592f7c184e2

### CODE REVIEW

File: src/transbank_webpay_rest/composer.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
     "require": {
-        "transbank/transbank-sdk": "2.0.8"
+        "transbank/transbank-sdk": "~2.0"
```
Comment (mastudillot) [inline src/transbank_webpay_rest/composer.json:3]: This must be two commits

File: .sonarcloud.properties [hunk 2]
```diff
@@ -0,0 +1 @@
+sonar.exclusions=docker-virtuemart3, src/transbank_webpay_rest/library/log4php/**
```
Comment (mastudillot) [inline .sonarcloud.properties:1]: Same, this must be two commits

**Review by mastudillot — CHANGES_REQUESTED** [review]
Separate the commits from the files where I left a comment

File: .sonarcloud.properties [hunk 3]
```diff
@@ -0,0 +1 @@
+sonar.exclusions=docker-virtuemart3
```
Comment (victormendoza96) [inline .sonarcloud.properties]: missing empty line

File: .sonarcloud.properties (hunk 2 above)
Comment (victormendoza96) [inline .sonarcloud.properties]: empty line

**Review by victormendoza96 — APPROVED** [review]

**Review by tribiec — APPROVED** [review]
LGTM!

**Pull request merged**